doc/source/whats_new.rst

```diff
@@ -31,6 +31,9 @@ Changelog
 
     - Add new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked response across time by `Jean-Remi King`_
 
+    - Add support to append new channels from a list of Raw objects by `Chris Holdgraf`_
```

Comment:
Hey all - I'm finally done being super sick and would like to finish up
this PR. I'm writing tests for Raw/Epochs/AverageTFR/Evoked, but had a
question to run past you guys:

For Evoked, I ran into the same problem with a missing field in "info" that
I ran into when creating RawArray objects using "create_info". In that
case, the field "buffer_size_sec" was missing when you created a new info
object from scratch. When you tried to merge Info objects with _merge_info,
it expected that field and threw a KeyError. So, I just included that in
the set of fields and it worked fine.

Now I am reading in the test evoked data like this:

`evoked = read_evokeds(fname, condition=0)`

And I'm getting the same error. So I wanted to ask you guys: is
buffer_size_sec something that should *not* be in the info objects? Instead
of adding the field to the `create_info` function, should I have instead
removed that field from the `_merge_info` function?



On Fri, Jul 17, 2015 at 12:38 AM, Denis A. Engemann <
notifications@github.com> wrote:

> In doc/source/whats_new.rst
> <https://github.com/mne-tools/mne-python/pull/2302#discussion_r34846072>:
>
> > @@ -31,6 +31,9 @@ Changelog
> >
> >      - Add new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked response across time by `Jean-Remi King`_
> >
> > +    - Add support to append new channels from a list of Raw objects by `Chris Holdgraf`_
>
> +1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2302/files#r34846072>.
>